src/PythonProjectBootstrapper/package/{{ cookiecutter.__empty_dir }}/README.md

```diff
@@ -48,19 +48,16 @@ Download an executable for Linux, MacOS, or Windows to use the functionality pro
 1. Download the archive for the latest release [here]({{ cookiecutter.github_url }}/{{ cookiecutter.github_username }}/{{ cookiecutter.github_project_name }}/releases/latest); the files will begin with `exe.` and contain the name of your operating system.
 2. Decompress the archive
 
-{% if cookiecutter.sign_binaries %}
+{% if cookiecutter.minisign_public_key != 'none' %}

```

Comment:
Nit: once more :)

Reply:
Good catch. I will add this to a separate commit after the other changes go in.